samples/Boxplot sample.html

```diff
@@ -0,0 +1,281 @@
+﻿<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=10; IE=Edge" />
+    <title>Boxplot</title>
+    <link rel="stylesheet" type="text/css" href="../src/css/idd.css" />
+    <link rel="stylesheet" type="text/css" href="../src/css/IDDTheme.css" />
+    
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.min.js"></script>
+    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/rxjs/3.1.2/rx.lite.min.js"></script>
+    <script src="../dist/idd.js"></script>
+    <script type="text/javascript">
+        $(document).ready(function () {
+            var markers = InteractiveDataDisplay.asPlot("chart").get("markers");
+
+            var data2 = $("#npz_csv").html();
```

Comment:
IDD plots (incl. markers) can read data from the plot host element, see https://github.com/predictionmachines/InteractiveDataDisplay/blob/master/samples/Markers.html.

For information, there is InteractiveDataDisplay.readCsv method; so you shouldn't add custom implementation  of csv parser to each sample. See https://github.com/predictionmachines/InteractiveDataDisplay/blob/master/src/idd/idd.readers.js.